downloads/swan-lake-release-notes/swan-lake-preview8/RELEASE_NOTE.md

```diff
@@ -0,0 +1,519 @@
+
+A service declared using the service-declaration syntax cannot be referred to using an identifier anymore. Instead, you need to use an object constructor prepended by the `service` keyword to declare a service object or create a new instance of a service class.
+
+```ballerina
+service object { } hello = service object {
+    string message = "Hello";
+    resource function get sayHello() returns string {
+
+    }
+
+}
+listener http:Listener l = new(7000);
+string[] basePath = ["/"]; // service base path: /
+l.attach(hello, basePath);
+```
+
+```ballerina
+service class HelloService {
+    string message;
+    resource function get sayHello() returns string {
+
+    }
+
+    function init(string message) {
+        self.message = message;
+    }
+}
+var helo = new HelloService();
+listener http:Listener l = new(7000);
+string[] basePath = ["hello", "path"]; // service on "/hello/path"
+l.attach(hello, basePath);
+```
+
+##### Transactional Services
+
+Ballerina transaction capabilities are extended to services. Now, you can define transactional resource functions and transactional remote functions. These functions will be participants of global distributed transactions. Infection and agreement protocols are implemented based on the Ballerina distributed transaction protocol. 
+
+By defining services as participants, all services work as a single unit of work. If any of the services fails, the whole transaction will be reverted and if all the services are successfully called, the transaction will be completed and committed successfully. 
+
+**Defining a Transactional Resource Function of a Service**
+
+```ballerina
+transactional resource function get message(http:Caller caller, http:Request req) {
+    http:Response res = new;
+    callSomeService(res);        
+    callAnotherService(res);
+    checkpanic caller->respond(res);
+}
+```
+
+**Defining  the Transactional Remote Functions of a Client Object**
+
+
+```ballerina
+transactional remote function callMyFirstService() returns @tainted any|error {
+    return self.httpClient->get("/echo/message");
+}
+
+transactional remote function callMySecondService() returns @tainted any|error {
+    return self.httpClient->get("/echo/message");
+}
+```
+
+**Calling the Service**
+
+```ballerina
+transaction {
+    var res1 = client->callMyFirstService();
+   var  res2 = client ->callMySecondService();
+   var x = commit;
+    if (x is error) {
+        // error code
+    } else {
+        // success code
+    }
+}
+```                                                             
+
+#### Standard Library
+
+##### HTTP Module Changes
+
+###### Service Declaration
+
+- Basepath is removed from the `ServiceConfig`. Use the absolute resource path, which begins with `/` as the basePath. The `absolute-resource-path` is optional, which defaults to `/` when not specified.
```

Comment:
`absolute-resource-path` comes from the spec, right? Should we just use "absolute resource path" instead because we usually don't emphasize the grammar when it comes to stdlib examples?

Reply:
+1